lib/system/strmantle.nim

```diff
@@ -25,34 +25,38 @@ const digitsTable = "0001020304050607080910111213141516171819" &
   #       res.add $i
   #   doAssert res == digitsTable
 
-
-func digits10(num: uint64): int {.noinline.} =
-  if num < 10'u64:
-    result = 1
-  elif num < 100'u64:
-    result = 2
-  elif num < 1_000'u64:
-    result = 3
-  elif num < 10_000'u64:
-    result = 4
-  elif num < 100_000'u64:
-    result = 5
-  elif num < 1_000_000'u64:
-    result = 6
-  elif num < 10_000_000'u64:
-    result = 7
-  elif num < 100_000_000'u64:
-    result = 8
-  elif num < 1_000_000_000'u64:
-    result = 9
-  elif num < 10_000_000_000'u64:
-    result = 10
-  elif num < 100_000_000_000'u64:
-    result = 11
-  elif num < 1_000_000_000_000'u64:
-    result = 12
-  else:
-    result = 12 + digits10(num div 1_000_000_000_000'u64)
+func digits10(x: uint64): int {.inline.} =
+  if x >= 10000000000'u64: # 1..10, 11..20
```

Comment:
Nitpicking: Can you use thousand separators among the number to make it more clear?

Reply:
done